parser_methods.py

```diff
@@ -443,75 +477,79 @@ def _pdf_to_text(action_result, pdf_file):
         return action_result.set_status(phantom.APP_ERROR, "Failed to parse pdf: {0}".format(error_text)), None
 
 
-def _docx_to_text(action_result, docx_file):
-    """docx is literally a zip file, and all the words in the document are in one xml document
-    doc does not work this way at all
-    """
-    WORD_NAMESPACE = "{http://schemas.openxmlformats.org/wordprocessingml/2006/main}"
-    PARA = WORD_NAMESPACE + "p"
-    TEXT = WORD_NAMESPACE + "t"
-
+def _docx_to_text(action_result: "ActionResult", docx_file: str) -> tuple[bool, Optional[str]]:
```

Comment:
This function contains the only real functional change in this PR. Other changes are all in support of type hinting all functions